Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/Status.hs

```diff
@@ -103,7 +104,9 @@ checkIfInVerification :: Id SP.Person -> Image.ImageType -> Flow ResponseStatus
 checkIfInVerification driverId docType = do
   verificationReq <- IVQuery.findLatestByDriverIdAndDocType driverId docType
   images <- IQuery.findRecentByPersonIdAndImageType driverId docType
-  onboardingTryLimit <- asks (.driverOnboardingConfigs.onboardingTryLimit)
+  person <- runInReplica $ QPerson.findById driverId >>= fromMaybeM (PersonNotFound driverId.getId)
```

Comment:
we are already fetching person data at top. Pass that only instead of fetching again.